wdl2cwl/main.py

```diff
@@ -60,10 +61,66 @@ def load_wdl_objects(
             return self.load_wdl_task(obj)
         raise Exception(f"Unimplemented type: {type(obj)}: {obj}")
 
+    def load_wdl_workflow(self, obj: WDL.Tree.Workflow) -> cwl.Workflow:
+        """Load WDL workflow and convert to CWL."""
+        inputs: List[cwl.WorkflowInputParameter] = []
+        outputs: List[cwl.WorkflowOutputParameter] = []
+        wf_steps: List[cwl.WorkflowStep] = []
+        wf_name = obj.name
+        wf_description = obj.meta["description"]
+        for call in obj.body:
+            call_name = call.name  # type: ignore
+            callee = call.callee  # type: ignore
+            namespace, _ = call.callee_id  # type: ignore
+            cwl_call_inputs = self.get_cwl_task_inputs(callee.inputs)
+            wf_step_inputs = [
+                cwl.WorkflowStepInput(id=x.id, source=f"{namespace}.{call_name}.{x.id}")
+                for x in cwl_call_inputs
+            ]
+            inputs.extend(
+                [
+                    cwl.WorkflowInputParameter(
+                        id=f"{namespace}.{call_name}.{x.id}",
+                        type=x.type,
+                        default=x.default,
+                    )
+                    for x in cwl_call_inputs
```

Comment:
Ah! I think this is an excellent case for when not to use list comprehensions :nerd_face: 

You are iterating `cwl_call_inputs`, and `cwl_call_ouputs` twice to build the outputs and parameters. Instead, with a `for` loop you can easily iterate it once and populate both lists :+1: 

I believe we could even do that in one go with list comprehensions too, but that wouldn't be easy to read & maintain I think.